ios/framework/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.4.0</string>
```

Comment:
Not really :) We could fix it to be completely clear but its not a technical issue.